protocol/0066-VALW-validator_tendermint_weights.md

```diff
@@ -15,37 +15,67 @@ If the network has no stake at all, then all validators would have equal voting
 
 ### Basic sanity check (<a name="0066-VALW-001" href="#0066-VALW-001">0066-VALW-001</a>)
 1. set up a network with 5 validators 
-1. give each of the validators the following number of self-staked tokens: `2000`
-1. wait for the delegation to become active in the next epoch
-1. when the epoch begins verify that the delegation went through
-1. check that Tendermint weights for the validators are `2000`
+2. give each of the validators the following number of self-staked tokens: `2000`
+3. wait for the delegation to become active in the next epoch
+4. when the epoch begins verify that the delegation went through
+
 
 ### Non-uniform stake check (<a name="0066-VALW-002" href="#0066-VALW-002">0066-VALW-002</a>)
 1. set up a network with 5 validators
-1. give the first `4` validators `1000` of self-stake each. Give the last validator `500`. 
-1. wait for the delegation to become active in the next epoch
-1. check that Tendermint weights for the first `4` validators are `2222` and the last validator `1111`. 
+2. give the first `4` validators `1000` of self-stake each. Give the last validator `500`. 
+3. wait for the delegation to become active in the next epoch
+
 
 ### Zero stake check  (<a name="0066-VALW-003" href="#0066-VALW-003">0066-VALW-003</a>)
 1. set up a network with 5 validators
-1. give the first `4` validators `1000` of self-stake each. Give the last validator `0`. 
-1. wait for the delegation to become active in the next epoch
-1. check that Tendermint weights for the first `4` validators are `2500` and the last validator `1`. 
+2. give the first `4` validators `1000` of self-stake each. Give the last validator `0`. 
+3. wait for the delegation to become active in the next epoch
+
 
 ### Changes to delegation during the epoch are reflected in the next epoch’s voting power (<a name="0066-VALW-004" href="#0066-VALW-004">0066-VALW-004</a>)
 1. set up a network with 5 validators
-1. give the first `4` validators `1000` of self-stake each. Give the last validator `0`. 
-1. wait for the delegation to become active in the next epoch
-1. check that Tendermint weights for the first `4` validators are `2500` and the last validator `1`. 
-1. just before the epoch ends the last validator self-stakes `500`. 
-1. next epoch starts 
-1. we see that Tendermint weights for the first `4` validators are `2222` and the last validator `1111`. 
+2. give the first `4` validators `1000` of self-stake each. Give the last validator `0`. 
+3. wait for the delegation to become active in the next epoch
+5. check that Tendermint weights
+6. just before epoch 0 ends the last validator self-stakes `500`. 
+7. epoch 1 starts 
+5. check that Tendermint weights
```

Comment:
What should be the outcome of the "check"? We can always "check" the weights - they will be an integer. The test should verify they're the right integer.